packages/common/src/actions/transactionQueue.ts

```diff
@@ -0,0 +1,14 @@
+import type { Transport, Chain, Account, WalletActions, WalletClient } from "viem";
+import { writeContract as mud_writeContract } from "../writeContract";
+import { sendTransaction as mud_sendTransaction } from "../sendTransaction";
+
+export const transactionQueue = <TChain extends Chain, TAccount extends Account>(): ((
```

Comment:
nit: slight preference for the `export function transactionQueue` definition style